Hardware/Hardware/CardReader/CardReaderServiceError.swift

```diff
@@ -209,6 +222,10 @@ extension UnderlyingError {
             return false
         }
     }
+
+    public var isLocationIDMissingError: Bool {
```

Comment:
Not totally sure we need this for a single case? We might need to add `.storePostcodeMissing` for #5320, in which case it could be helpful, but I could also see that error being collapsed into this one. Regardless, we'll need to unwrap and downcast at the call site so that we can get the `adminUrl` out when that's included

Reply:
Maybe a more generic name like `isLocationError`? It seems like marking these special cases clearly simplifies the client code a lot.